.travis.yml

```diff
@@ -0,0 +1,23 @@
+dist: trusty
+sudo: false
+
+jobs:
+  include:
+    - state: Angular tests
+      language: node_js
+      node_js:
+        - '13'
+      cache:
```

Comment:
no need for caching, time save is not worth it, and can lead to version conflicts after changes in dependencies list

Reply:
Done